addons/fluxcd/resources/components/source-controller.cue

```diff
@@ -6,6 +6,11 @@ controllerArgs: [...]
 _targetNamespace: string
 _sourceControllerName: "fluxcd-source-controller"
 
+imageControllerDefaultArgs: controllerArgs + [
+					"--storage-path=/data",
+					"--storage-adv-addr=http://" + _sourceControllerName + "." + _targetNamespace + ".svc:9090",
+]
+
```

Comment:
Why we need add these config?

Reply:
well, these config were not added by me: [link](https://github.com/kubevela/catalog/blame/858842fb9974a25e515d66ffde4fef49bdaad871/addons/fluxcd/resources/components/source-controller.cue#L81)